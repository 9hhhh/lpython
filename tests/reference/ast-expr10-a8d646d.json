{
    "basename": "ast-expr10-a8d646d",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/expr10.py",
    "infile_hash": "9089a16432d23b099036b034bda51ca568f9bf709e55e2c23d1c2fcc",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-expr10-a8d646d.stdout",
    "stdout_hash": "f2eba3c9c330548e101fbca98c793daa2c64edfb05f1050f8aea1f2e",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}