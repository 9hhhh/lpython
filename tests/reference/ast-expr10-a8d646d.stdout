(Module [(FunctionDef test_UnaryOp ([] [] [] [] [] [] []) [(AnnAssign (Name a Store) (Name i32 Load) () 1) (Assign [(Name a Store)] (UnaryOp UAdd (ConstantInt 4 ())) ()) (Assign [(Name a Store)] (UnaryOp USub (ConstantInt 500 ())) ()) (Assign [(Name a Store)] (UnaryOp Invert (ConstantInt 5 ())) ()) (Assign [(Name a Store)] (UnaryOp Not (ConstantInt 5 ())) ()) (Assign [(Name a Store)] (UnaryOp Not (UnaryOp USub (ConstantInt 1 ()))) ()) (Assign [(Name a Store)] (UnaryOp Not (ConstantInt 0 ())) ()) (AnnAssign (Name f Store) (Name f32 Load) () 1) (Assign [(Name f Store)] (UnaryOp UAdd (ConstantFloat 1.000000 ())) ()) (Assign [(Name f Store)] (UnaryOp USub (ConstantFloat 183745.534000 ())) ()) (AnnAssign (Name b1 Store) (Name bool Load) () 1) (AnnAssign (Name b2 Store) (Name bool Load) () 1) (AnnAssign (Name b3 Store) (Name bool Load) () 1) (Assign [(Name b1 Store)] (ConstantBool .true. ()) ()) (Assign [(Name b2 Store)] (UnaryOp Not (ConstantBool .false. ())) ()) (Assign [(Name b3 Store)] (UnaryOp Not (Name b2 Load)) ())] [] () ())] [])
